lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -161,6 +161,9 @@ def __init__(self, module, host, port, protocol):
     def dnsrecord_find(self, zone_name, record_name):
         return self._post_json(method='dnsrecord_find', name=zone_name, item={'idnsname': record_name, 'all': True})
```

Comment:
could you alternatively use a ternary operation to change the method to dnsrecord_find within the local dnsrecord_find() function only if record_name=="@"?  Presumably this would be wanted for any attempt to access the '@' record... But I'll defer to those with more experience in this project and python.

Reply:
Yeah was considering renaming the method to something like get_dns_record() and adding the if in there like you mention, however, unsure on conventions or style in this repo in general. @Akasurde @fxfitz probably better positioned to advise on that.